src/core/enum-helpers.h

```diff
@@ -3,9 +3,13 @@
 
 #pragma once
 
+#include <src/basics.h>
 #include <librealsense2/hpp/rs_types.hpp>
 
 
+#define UNKNOWN_VALUE "UNKNOWN"
```

Comment:
This file is in `/src` and is included only by files from `/src`.
Doesn't it mean it's C++? Can we use `static const/expr` instead of `#define`?

Reply:
Added.